examples/ads.amp.html

```diff
@@ -715,6 +716,13 @@ <h2 id="zergnet">ZergNet</h2>
       type="zergnet"
       data-zergid="42658">
   </amp-embed>
+
+  <h2 id="zucks">Zucks</h2>
+  <amp-ad width="320" height="50"
+      type="zucks"
+      data-frame_id="_bda46cf5ac">
+  </amp-ad>
```

Comment:
please sync the file to HEAD of master, and follow the other examples to add the `placeholder` and `fallback` tags.